versions/0.0.0.md

```diff
@@ -0,0 +1,365 @@
+     * `metadata/depositor-info/message-from-depositor.txt`.
+
+4. **(SIP)** The `metadata` directory MAY contain a file called `metadata/depositor-info/message-from-depositor.txt`.
+
+5. The `metadata` directory MUST NOT contain any other files or directories.
+
+### 3 Metadata requirements
+
+#### 3.1 `metadata/dataset.xml`
+1. The file `metadata/dataset.xml` MUST adhere to [DANS dataset metadata schema].
+
+2. The file `metadata/dataset.xml` MAY have one `dcterms:license` element as a child of the
+   `dcmiMetadata` element. It MUST NOT have more than one such element. If present, and if it contains an attribute
+   `xsi:type="dcterms:URI"`, then the element text MUST be one of the [URIs of approved licenses].
+
+3. **(AIP)** The file `metadata/dataset.xml` MUST contain at least one `dcterms:identifier` element with
+   an attribute `xsi:type="id-type:DOI"`. The text of this element MUST be a syntactically valid [DOI] identifier
+   which SHOULD be resolvable.
+
+4. Any `dcx-dai:DAI` elements MUST contain a syntactically valid [Digital Author Identifier] (DAI) with a valid check digit.
+   This DAI SHOULD refer to an existing author. The DAI MAY be provided as a `info:eu-repo/dai/nl/` URI.
+
+5. Any `gml:postList` elements nested in `gml:Polygon` elements:
+
+   * MUST have an even number of values.
+   * MUST have at least three different pairs of values, each of which describes one point.
+   * MUST start and end with the same pair of values.
+
+6. If a `gml:MultiSurface` is used, all nested `gml:Polygon` MUST have the same `srsName` attribute.
+
+7. Any `gml:Point`, `gml:lowerCorner` and `gml:upperCorner` elements MUST have at least two values.
+
+8. Any `dc:identifier` or `dcterms:identifier` element with `xsi:type="id-type:ARCHIS-ZAAK-IDENTIFICATIE`
+   MUST have a value of 10 or fewer characters.
+
+
+#### 3.2 `metadata/files.xml`
+1. If the file `metadata/files.xml` declares the [DANS bag file metadata schema] namespace then it MUST
+   adhere to that schema.
+
+2. The document element MUST be `files`.
+
+3. The document element MUST contain zero or more `file` elements, and MUST NOT contain other elements.
+   <!-- Skip if XSD is used -->
+
+4. Each `file` element MUST have a `filepath` attribute which contains the bag local path to the payload file described.
+   Directories and non-payload files MUST NOT be described by a `file` element.
+
+5. There MUST NOT be more than one `file` element corresponding to a payload file and every payload file MUST be described
+   by a `file` element.
+
+6. Each `file` element MUST contain at least one `dcterms:format` element containing the [Media Type]
+   of the file described.
+
+7. Each `file` element MAY contain any number of other [DC] or [DCTERMS] elements describing the file. It MUST NOT contain elements
+   from other namespaces, unless permitted by the [DANS bag file metadata schema] schema.
+   <!-- Skip if XSD is used -->
+
```

Comment:
new rule 3.2.8 from EASY-2029, do you want it here already? or in a separate PR?

3.2.8
Each `file` element MAY contain either one dcterms:accessRights (DEPRECATED) or files:accessibleToRights and/or files:visibleToRights (RECOMMENDED). The values of these elements are limited to ANONYMOUS, RESTRICTED_REQUEST and NONE. It is recommended to make files visible to all, by selecting `files:visibleToRights=ANONYMOUS`